lib/dialects/postgres/connection-manager.js

```diff
@@ -119,6 +119,7 @@ ConnectionManager.prototype.disconnect = function(connection) {
   return new Promise(function (resolve, reject) {
     connection.end();
     resolve();
+    return null;
```

Comment:
Attempting to squash warnings - http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-created-in-a-handler-but-none-were-returned-from-it